components/draw/draw.service.ts

```diff
@@ -171,7 +184,7 @@ export class HsDrawService {
         visible: true,
         removable: true,
         editable: true,
-        synchronize: true,
```

Comment:
What if we set this to true if and only if we have layman endpoint defined?

Reply:
7cc8eaf00c19caaad4a9bb9d5643cb06ffa1d422